libraries/AP_Frsky_Telem/AP_Frsky_Telem.cpp

```diff
@@ -162,6 +169,13 @@ void AP_Frsky_Telem::passthrough_wfq_adaptive_scheduler(uint8_t prev_byte)
                 case 8:
                     packet_ready = AP::battery().num_instances() > 1;
                     break;
+                case 11:
+                    if (_rpm_sensor == nullptr) {
+                        packet_ready = false;
+                    } else {
+                        packet_ready = (_rpm_sensor->enabled(0) || _rpm_sensor->enabled(1));
```

Comment:
This LGTM. We'll be able to send the rpm on the passthru if either sensor is enabled. Does this still allow for use of external sensors for engine(s)? In heli we use it for rotor rpm for governor, and runup_complete() (HeliPilot only - this never got into ArduPilot). And the plan is to add a critical headspeed warning alarm (MavLink message) to get the pilot's attention in event of engine or drivetrain failure in Auto flight mode. Having this in the passthru makes it a viable reality since the latency to a GCS is too much to make it usable.

Reply:
> Does this still allow for use of external sensors for engine(s)?
external as wired on the s.port bus than yes, those do not interfere with passthrough.